python_apps/pypo/liquidsoap/1.3/fdkaac.liq

```diff
@@ -0,0 +1,24 @@
+            if bitrate == 24 then 
```

Comment:
Can we not do something like bitrate=bitrate in the script?

Reply:
I'm not sure how much of fdk-aac is still working.

Reply:
We can't really `bitrate=bitrate` because this also encodes info on which profile to activate and which feature like afterburner and sbr to activate. This "matrix" here encodes the possible combinations as there are rules in the standard as to which bitrate you get to use with a profile and what features you are allowed to activate.

Reply:
Why I ask is that this matrix seems large and is repeated across a couple of files. It would be nice to be able to reduce that complexity

Reply:
Untangling this results in some code that only shaves off maybe 2-3 lines off of the definitions, mostly because of [how liquidsoap handles format "variables"](https://www.liquidsoap.info/doc-1.4.3/encoding_formats.html#format-variables-or-lack-of-rather..) making most of it moot.

The previous code used to share `fdkaac.liq` and other encoders across versions. I decided to create copies of the encoders since it felt more future proof and they are more or less static anyway.

Reply:
Ok, cool